docs/tracing.md

```diff
@@ -0,0 +1,83 @@
+# Tracing
+
+containerd supports OpenTelemetry tracing since v1.6.0.
+Tracing currently targets only gRPC calls.
+
+## Sending traces from containerd deamon
+
+By configuring `io.containerd.tracing.processor.v1.otlp` plugin.
+containerd daemon can send traces to the specified OpenTelemetry endpoint.
+
+```toml
+[plugins."io.containerd.tracing.processor.v1.otlp"]
+    endpoint = "http://localhost:4318"
```

Comment:
I wonder if this may break users setting HTTP_PROXY for registries like commented here: https://github.com/containerd/containerd/issues/1990#issuecomment-356780876

Are there any other http dependencies of Containerd or this is the only one after registries?

Reply:
Do you mean it may break the proxy setting for those users or that those users may end up with this HTTP traffic unexpectedly going through their proxy?

Reply:
second. They configured it for registry and now getting new traffic for telemetry

Reply:
Don't think that should be a big concern. At least for me, that is ideal for debugging. It is something to be aware of if you configure that and turn this on

Reply:
perhaps this can be added to the release notes. I agree on priority

Reply:
It would be better to document HTTP_PROXY and other environment variables, outside of this tracing doc because all of containerd code honor that. IMHO Go itself doesn't do a great job though.

https://pkg.go.dev/net/http#ProxyFromEnvironment